modules/core/src/main/java/org/apache/ignite/spi/systemview/view/ContinuousQueryView.java

```diff
@@ -44,6 +44,9 @@
     /** Routine id. */
     private final UUID routineId;
 
+    /** Class name of the local listener */
+    private String localListener;
```

Comment:
Let's also add a consistent ID of the node that has registered the CQ (the one that is running the local listener)

Reply:
Ok, done!